README.md

```diff
@@ -493,6 +497,10 @@ This isn't necessary at this stage, but it seems wise to note the intended patte
 
 The GH actions builds use the Knapsack-pro gem to reduce build time by parallelizing the RSpec and Cucumber tests. There is no need to regenerate the knapsack_rspec_report.json file, Knapsack Pro will dynamically allocate tests to ensure tests finish as close together as possible.
 
+#### Chromedriver issues
+
+If you encounter CI failures for cucumber and rspec feature tests it may be caused by our CI GitHub runner being on the latest version of Chrome while our testing libraries are yet to support it. To fix this, you can pin the version of Chrome in the CI to an older, known working version. This can be done in the `.github/workflows/ruby_test.yml` workflow under `Setup stable Chrome`. If this is required ensure you revert the change once the testing libraries are updated.
+
```

Comment:
Is there a URL with ref to the versions of Chrome? For example, where did `128` come from? Sorry, just trying to understand, so should it be `chrome-version: latest` normally, but update `chrome-version: xx` when wanting to pin the version?

Reply:
> @BenTopping Is there a URL with ref to the versions of Chrome? For example, where did `128` come from? Sorry, just trying to understand, so should it be `chrome-version: latest` normally, but update `chrome-version: xx` when wanting to pin the version?

128 came from the recommended fix on the capybara page, not sure on the significance, maybe its the last version without the breaking changes. Chrome versions can be found by somewhere like https://developer.chrome.com/release-notes. Yes `chrome-version: latest` should be the normal.